modxcms/revolution
# Fix grouping grid sorting - ACL grids

### What does it do?
Adds new grouping logic to ACL GetList processors and config props in the corresponding Ext grid classes.

### Why is it needed?
Aside from the Settings grid,* other grouping grids (such as the ACL ones) neither group properly, nor sort within groups properly.

### How to test

1. Rebuild may be necessary and compress_js must be off. 
2. Create a variety of ACL entries so there are at least a few main items to group (this is the Minimum Role, which is what all ACLs are grouped by as a default [can be changed in the column menu]), and a few entries within those groups (meaning you'll need multiple contexts, namespaces, resource groups, media sources, and element categories created ahead of time to assign these roles to).
3. Verify that items are grouped as expected.
4. Verify that sorting on each column works as expected.

### Related issue(s)/PR(s)
No related issue.

* The settings grid does not group correctly if the grouping field is changed. That will be addressed in a different PR.
